cluster-up/cluster/kind/common.sh

```diff
@@ -39,7 +39,13 @@ ETCD_IN_MEMORY_DATA_DIR="/tmp/kind-cluster-etcd"
 
 function _wait_kind_up {
     echo "Waiting for kind to be ready ..."
-    while [ -z "$(docker exec --privileged ${CLUSTER_NAME}-control-plane kubectl --kubeconfig=/etc/kubernetes/admin.conf get nodes --selector=node-role.kubernetes.io/master -o=jsonpath='{.items..status.conditions[-1:].status}' | grep True)" ]; do
+    # For providers with version > 1.19 we use control-plane node role, master otherwise
```

Comment:
i think we don't need please the comment, the code would be enough, (and soon to be outdated)